docs/maatregelen/0-org-05-bestaande-governance.md

```diff
@@ -0,0 +1,66 @@
+- [privacygovernance](https://www.cip-overheid.nl/media/eeqkauey/20200814-handleiding-privacy-governance-v3_2.pdf)
+
+In veel organisaties werken privacy- en informatiebeveiliging nauw samen van strategisch organisatieniveau tot operationeel omdat deze onderwerpen raken aan beide domeinen. 
+Dit geldt in nog grotere mate voor vraagstukken rondom de inzet van algoritmes, die vragen om samenwerking en expertise vanuit veel verschillende invalshoeken.
+Communicatie en vindbaarheid tussen de verschillende domeinen is hierin essentieel. 
+De eindverantwoordelijkheid voor algoritmes moet bij één entiteit belegd worden, die let op het betrekken van de juiste (interne) partijen. 
+
+Bij algoritmegovernance moet rekening worden gehouden met mogelijk conflicterende belangen. 
+Bijvoorbeeld, het kan voorkomen dat de business of IT-teams innovatie willen nastreven, terwijl compliance teams juist vragen om het afremmen van innovatie. 
+Een algortime-eindverantwoordelijke dient hierin de afwegingen te maken, in samenspraak.
+Zorg voor [bestuurlijke betrokkenheid](0-org-04-politiek-bestuurlijke-verantwoordelijkheid.md) bij het maken van deze afwegingen. Een strategie of visie zoals vastgelegd in een meerjarenbeleidsplan helpt sturing te geven bij het maken van afwegingen.
+
+## Aandachts- en actiepunten
+* Welke lessen zijn geleerd met de implementatie van de AVG?
+* Is er iemand intern verantwoordelijk gemaakt voor (toezicht op) algoritmes?
+* Hoe communiceren experts vanuit verschillende onderwerpen zoals privacy, informatiebeveiliging en data, die het onderwerp algoritme raken, op dit moment?
+* Het [opstellen van RACI-model](2-owp-01-rollen-en-verantwoordelijkheden.md) helpt bij het in kaart brengen van de samenhang tussen verschillende governancestructuren.
+* Hoe worden afwegingen gemaakt in de organisatie tussen innovatie en compliance?
+* Een orgaan als een ethische commissie (AI) kan helpen te verbinden en keuzes te maken. Let wel op dat dit niet een 'praatgroep' wordt zonder daadwerkelijk mandaat en draagvlak.
+
+
+## Voorbeelden
+Een concreet voorbeeld van samen optrekken is [deze handreiking](https://www.cip-overheid.nl/media/av0dmahv/20230614-gezamenlijk-gebruik-iama-en-model-dpia-rijksdienst-v1-0.pdf) om de IAMA en DPIA gezamelijk uit te voeren.
+
+
+!!! example "Voorbeeld: Ervaringen Kadaster" 
+
+    1) Bij inwerkingtreding van de AVG ging men snel aan de slag met DPIA’s, vinkenlijstjes. Let op voor een schijndossier: Begrijp wat er toegepast moet worden, voordat je aan de slag gaat; niet uitvoeren om maar te laten zien dat je eraan voldoet.
+    2) Nadenken over governance. De FG is bij de wet geregeld, maar er is meer nodig. Welke profielen heeft de ondersteuning voor de uitvoering; (centrale) privacy officers en is hier ruimte voor?
+    3) De FG kijkt naar de wet en interpreteert dit. Zorg dat deze interpretatie gelijk is binnen de organisatie en de collega’s die binnen de governance structuur samenwerken.
```

Comment:
Een FG neemt meerdere zaken mee in zijn of haar beoordeling. Dit is daarom onjuist.